pkg/api/v1/helpers.go

```diff
@@ -454,3 +455,70 @@ type NodeResources struct {
 	// Capacity represents the available resources of a node
 	Capacity ResourceList `protobuf:"bytes,1,rep,name=capacity,casttype=ResourceList,castkey=ResourceName"`
 }
+
+func (rc *ReplicationController) DecodeNestedObjects(d runtime.Decoder) error {
```

Comment:
@smarterclayton , any suggestion on what's original intention of `runtime.NestedObjectDecoder`? Did not find example implementation in master branch.

I'll move it (e.g. decode.go) when we got finial decision :).